utils/DeployedContractsHelpers.ts

```diff
@@ -138,6 +138,19 @@ export async function getAllBDStablePools(hre: HardhatRuntimeEnvironment): Promi
   return allStablePools;
 }
 
+export async function getAllExistingBDStablePools(hre: HardhatRuntimeEnvironment): Promise<BdStablePool[]> {
```

Comment:
right above this function there is a function `getAllBDStablePools` - isn't it a duplication?